__tests__/integration.test.js

```diff
@@ -167,7 +171,95 @@ describe("/api/articles/:article_id/comments", () => {
 				.get("/api/articles/invalid/comments")
 				.expect(400)
 				.then(({ body: { msg } }) => {
-					expect(msg).toBe("Invalid Input");
+					expect(msg).toBe("Invalid id");
+				});
+		});
+	});
+	describe("POST", () => {
+		test("POST:201 adds a new comment to the specified article and sends the comment back to the client", () => {
+			const expectedComment = {
+				comment_id: 19,
+				body: "Fugiat molestiae iure et qui consequatur expedita quia. Est sed repellat nesciunt nulla sit in dolor laudantium. Totam vero et quam. In numquam magnam voluptas itaque. Quisquam vel vitae doloribus vel id laboriosam quibusdam.",
+				article_id: 4,
+				author: "rogersop",
+				votes: 0,
+				created_at: expect.toBeString(),
+			};
+			return request(app)
+				.post("/api/articles/4/comments")
+				.send({
+					username: "rogersop",
+					body: "Fugiat molestiae iure et qui consequatur expedita quia. Est sed repellat nesciunt nulla sit in dolor laudantium. Totam vero et quam. In numquam magnam voluptas itaque. Quisquam vel vitae doloribus vel id laboriosam quibusdam.",
+				})
+				.expect(201)
+				.then(({ body: { postedComment } }) => {
+					expect(postedComment).toMatchObject(expectedComment);
+					return request(app).get("/api/articles/4/comments").expect(200);
+				})
+				.then(({ body: { comments } }) => {
+					expect(comments).toContainEqual(expectedComment);
+				});
```

Comment:
I'd add in another 201 test to check that it ignores any unnecessary properties on the request body 👍